docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Update KubeLB integration to support v1.1.0 ([#13661](https://github.com/kubermatic/kubermatic/pull/13661))
+    - **ACTION REQUIRED:** [EE] If you are using KubeLB, before upgrading to KKP 2.26, please upgrade KubeLB management cluster to [v1.1.0](https://docs.kubermatic.com/kubelb/v1.1/installation/management-cluster). This is required for KKP integration of KubeLB to be functional.
+    - KubeLB: KubeLB integration has been upgraded to support KubeLB v1.1
+    - KubeLB: Options to enable Gateway API and Load Balancer class have been added at seed and cluster level
+- Automated migration from machine-controller user data to OSM ([#13659](https://github.com/kubermatic/kubermatic/pull/13659))
+    - KKP will perform automated migrations for clusters that are using machine-controller user data to OSM
+    - **ACTION REQUIRED:** Migration from machine-controller user data to OSM is automated. Users can scale up/down their machines, and there won't be any hindrance. However, existing machines/nodes using MC user data will not be rotated. This is by design to avoid unnecessary node rotations, but this can also lead to a drift between the cloud-config for new and old machines. It is recommended, not mandatory, to either rotate the machines one by one or rotate the machine deployment as a whole following https://docs.kubermatic.com/kubermatic/v2.26/cheat-sheets/rollout-machinedeployment/ 
+
+### API Changes
+
+- Bump Seed MLA Loki and Promtail ([#13281](https://github.com/kubermatic/kubermatic/pull/13281))
+    - Update Seed-MLA Loki to 2.9.6; this Helm chart version now uses a slightly different configuration syntax, please change `.loki.config` into `.loki.loki`.
+    - Update Seed-MLA Promtail to 2.9.3
+- Add `spec.componentsOverride.operatingSystemManager` to allow overriding OSM settings and resources ([#13285](https://github.com/kubermatic/kubermatic/pull/13285))
+- Loadbalancer provider (lb-provider) & loadbalancer method (lb-method) can be configured at the datacenter for openstack provider ([#13574](https://github.com/kubermatic/kubermatic/pull/13574))
+- Operating System Manager is now mandatory to create a functional cluster since machine-controller user-data plugins have been removed(https://github.com/kubermatic/machine-controller/pull/1789). Thus, the Operating System Manager is now always enabled for the user clusters ([#13381](https://github.com/kubermatic/kubermatic/pull/13381))
+- Separate container image tag/tag-suffix can be set for KKP UI & KKP API.**ACTION REQUIRED**: If custom image tag/tag-suffix is being used for KKP UI & the admin desires to use the same (or different) custom tag/tag-suffix for the Kubermatic API image as well, then it needs to be explicitly set in the `KubermaticConfiguration.spec.api.dockerTag/dockerTagSuffix` otherwise the default tag for the KKP version will be used ([#13274](https://github.com/kubermatic/kubermatic/pull/13274))  
```

Comment:
ACTION REQUIRED is squished in there with no spaces and also shouldn't the action required bit be listed in the action required section?